packages/CollapsibleChecklists/src/CollapsibleChecklists.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Heading from "./components/Heading";
```

Comment:
should import paprika components instead of relative paths

Reply:
oh its not a paprika Heading, its my own

Reply:
Wondering if perhaps we should try to not call our own components the same as paprika component names in general to avoid that confusion